index.js

```diff
@@ -14,3 +68,47 @@ navLinks.forEach((link) => {
     sidebar.classList.remove('show-sidebar');
   });
 });
+openModal.forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    const projectName = e.target.parentElement.parentElement.dataset.name;
+    const { title, info, tech, src, live, sourceCode } = projects.find(
+      (item) => item.name === projectName
+    );
+    const modalContent = `<h3 class="card-title">
+                  ${title}
+                </h3>
+                <ul class="tech-container">
+                ${tech.map((tech) => `<li class="tech">${tech}</li>`)}
+                </ul>
+                <div class="project-card">
+                  <div class="pop-img-container">
+                    <img
+                      src="${src}"
+                      alt="${title}"
+                      class="card-header-img"
```

Comment:
- [ ] Good job with the design of the modal window 👏 Kindly add a dark overlay behind the modal with a slight blur (for both desktop and mobile view) as shown in the Figma template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/208717490-d788db8d-204f-4d61-9578-3725d5b1bdec.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/208717526-1462be0b-a8cb-4d7c-8769-c5907d23acef.png)